examples/plot_heterosc_estimation_modelling.py

```diff
@@ -26,28 +28,24 @@
 import geoutils as gu
 
 # %%
-# We start by loading example files including a difference of DEMs at Longyearbyen, the reference DEM later used to derive
-# several terrain attributes, and the outlines to rasterize a glacier mask.
-# Prior to differencing, the DEMs were aligned using :ref:`coregistration_nuthkaab` as shown in
-# the :ref:`sphx_glr_auto_examples_plot_nuth_kaab.py` example. We later refer to those elevation differences as *dh*.
+# Here, we detail the steps used by ``xdem.spatialstats.infer_heteroscedasticity_from_stable`` exemplified in
+# :ref:`sphx_glr_auto_examples_plot_infer_heterosc.py`. First, we load example files and create a glacier mask.
 
 ref_dem = xdem.DEM(xdem.examples.get_path("longyearbyen_ref_dem"))
 dh = xdem.DEM(xdem.examples.get_path("longyearbyen_ddem"))
 glacier_outlines = gu.Vector(xdem.examples.get_path("longyearbyen_glacier_outlines"))
 mask_glacier = glacier_outlines.create_mask(dh)
 
 # %%
-# We use the reference DEM to derive terrain variables such as slope, aspect, curvature (see :ref:`sphx_glr_auto_examples_plot_terrain_attributes.py`)
-# that we'll use to explore potential non-stationarities in elevation measurement error
-
-# We compute the slope, aspect, and both plan and profile curvatures:
+# We derive terrain attributes from the reference DEM (see :ref:`sphx_glr_auto_examples_plot_terrain_attributes.py`),
+# which we will use to explore the variability in elevation error.
 slope, aspect, planc, profc = \
```

Comment:
Can't we pass `ref_dem` instead of `ref_dem.data` and get rid of `resolution`?

Reply:
Remove resolution, but had to add `gu.spatial_tools.get_array_and_mask()[0]` below. We need to discuss the possible implementation of `get_array` as a `Raster` class method!

Reply:
In that case, the previous implementation was probably fine. I didn't realize the issue.